src/jit/lsraarmarch.cpp

```diff
@@ -403,7 +403,17 @@ void Lowering::TreeNodeInfoInitPutArgReg(
     info.srcCount++;
 
     // Set the register requirements for the node.
-    const regMaskTP argMask = genRegMask(argReg);
+    regMaskTP argMask = genRegMask(argReg);
```

Comment:
In looking through the code, I can't see where this method (`Lowering::TreeNodeInfoInitPutArgReg()`) is called. It appears that for arm64 it is handled with the default case, and for arm it is handled directly in `Lowering::TreeNodeInfoInit()`.